docs/content/2.deploy/3.custom-presets.md

```diff
@@ -5,7 +5,77 @@ icon: ri:file-code-line
 # Custom Preset
 
 ::alert{type="warning"}
-**Work in Progress**
+**Advanced**
 :br
-Documentation for this section is not yet complete.
+This is an advanced feature.
 ::
+
+If you want to use a provider that Nitro doesn't support, or want to modify an existing one, you can create a custom preset.
```

Comment:
We need to clarify it is for "local" presets (preset is not an external package but is in the same project)